<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose_enemy</class>
 <widget class="QDialog" name="choose_enemy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="monster3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="monster1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="monster2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="wallpaper">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="monster4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <zorder>wallpaper</zorder>
  <zorder>monster3</zorder>
  <zorder>push3</zorder>
  <zorder>push1</zorder>
  <zorder>push2</zorder>
  <zorder>monster1</zorder>
  <zorder>monster2</zorder>
  <zorder>monster4</zorder>
  <zorder>push4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
